grails-app/services/org/pih/warehouse/picking/PickTaskService.groovy

```diff
@@ -72,6 +74,15 @@ class PickTaskService {
                 }
             }
 
+            if (command.requisitionId) {
+                createAlias("requisition", "r")
+
+                or {
+                    eq("r.id", command.requisitionId)
+                    eq("r.requestNumber", command.requisitionId)
```

Comment:
Probably not necessary since we'll have the requisition IDs right?

Reply:
yep, requestNumber is not required but I added it to make it more flexible (if ever needed searching by requisition number)